### DISCUSSION TITLE
manual cell type annotation using dot plot and violin plot

### DISCUSSION THREAD

**Discussion by dahun73** [satijalab/seurat#5954] [Q&A] [answered]
Hello :)

For annotating cell type manually, I have some questions.

In my case, some clusters are ambiguous to annotate. Markers genes from 'FindMarkers' have only two positive genes for these clusters. But these two marker genes are not specific markers genes(I think this cluster may be intermediate state). So, I tried to use other marker genes in reference papers making violin and dot plot. 

I already know that I should use **'integrated' assay** when we try to find cell identity. In this ambiguous cluster, I can not see expression of some genes for 'integrated' assay. Actually warning message came out(Warning: Could not find 'OOO' in the default search locations, found in RNA assay instead).

In this case, is it okay to use 'RNA' assay? Because result of 'integrated' and 'RNA' assay is too different, I'm confused which cell type I should assign.

Thank you in advance!

**Comment by timoast** [comment, marked as answer]
For identifying differentially expressed genes and visualizing gene expression, you should always use the original data, _not_ the integrated data.
